docs/content/users/usage/architecture.md

```diff
@@ -102,6 +102,27 @@ Files beginning with `.` are hidden because they shouldn’t be fiddled with; mo
 
 There’s only one global `.ddev` directory, which lives in your home directory: `~/.ddev` (`$HOME/.ddev`).
 
+!!!tip "Where is my global `.ddev` config?"
+    Use `ddev config global` (look at `global-config-path`) to check which location is currently used for the `.ddev` global directory.
+
+!!!tip "What if I don't want to clutter up my `$HOME` with a `.ddev` directory?"
+    `~/.ddev` can be moved to `$XDG_CONFIG_HOME/ddev` directory if `$XDG_CONFIG_HOME` is set:
+
+    ```bash
+    # permanently set environment variable depending on your OS (e.g. in ~/.bashrc):
+    export XDG_CONFIG_HOME="/my-desired-path"
+    # restart the terminal and run:
+    mv ~/.ddev /my-desired-path/ddev
+    ```
+
+    Otherwise, if `$XDG_CONFIG_HOME` is not set, `~/.ddev` can be moved to:
```

Comment:
I didn't know about this feature, and this worries me a bit. I was expecting only two places, ~/.ddev and $XDG_CONFIG_HOME/ddev

Reply:
Okay, it makes sense, I will make it work with XDG_CONFIG_HOME only without and locations by default.

Reply:
The XDG base directory spec states:

> If $XDG_CONFIG_HOME is either not set or empty, a default equal to $HOME/.config should be used.

So I do think we should support `~/.config/ddev` by default if the variable is not set.

Things get murky for macOS and Windows, but the directories you've listed here seem to match convention.

Reply:
I previously added `TestGetXdgConfigHomeDir` specifically to test these different locations.

@danepowell, I think in case of Linux you are right.

Linux apps with support for XDG_CONFIG_HOME (see Notes column) to show the current situation with this topic:
https://wiki.archlinux.org/title/XDG_Base_Directory#Supported

Maybe I should just remove the default locations for macOS and Windows.